gensrc/proto/function_service.proto

```diff
@@ -35,8 +35,8 @@ message PFunctionCallRequest {
 }
 
 message PFunctionCallResponse {
-    optional PValues result = 1;
-    optional PStatus status = 2;
+    required PValues result = 1;
```

Comment:
why changing to requried?

Reply:
During the test, I found that forgetting to return would lead to be crash

Reply:
fixed